pkg/libfixmath/Makefile.dep

```diff
@@ -6,3 +6,9 @@ ifneq (,$(filter libfixmath-unittests,$(USEMODULE)))
   # The round function is not provided by the msp430 toolchain
   FEATURES_BLACKLIST += arch_msp430
 endif
+
+ifneq (,$(filter gnu gcc,$(TOOLCHAIN)))
```

Comment:
Now you could put this in a `Makefile.gnu.dep`.

Reply:
tests/pkg_libfixmath application doesn't fail on arm, just the unittests. So do we really want to blacklist this package for all ARM ? or just the application that is building the unittests ?

Reply:
Well, it's the division that is off, do you want to use the package if the division is not right? If you would want to use it still then I'm fine with blacklisting only in the application.

Reply:
I meant that it could be blacklisted above if the libfixmath-unittests is used. But you are right that this is a problem if the division is not correct.

Reply:
Initially, I had done as you suggested, but the unittest module only enable unittests to be run but the function is still there, and still, faulty whether the module is included o not (or at least that is what I expect)

Reply:
Well, updating the libfixmath package to the latest version fixes the problem on gcc: it works on samr21-xpro and b-l475e-iot01. I'll open a PR in a moment.
It seems the project has been active regularly during the last 2 years.